src/protocols/mqtt5/Mqtt5Protocol.cpp

```diff
@@ -200,7 +197,7 @@ bool Mqtt5Protocol::handleAck(const IStreamConnectionPtr& connection, unsigned i
     std::unique_lock<std::mutex> lock(m_mutex);
     if ((packetId != 0) && (packetId < m_messageIdsAllocated.size()))
     {
-        Mqtt5Command cmd = static_cast<Mqtt5Command>(command);
```

Comment:
two mqtt5command in single line :-)

Reply:
I usually try to avoid auto, so that the type is visible to the reader